pyproject.toml

```diff
@@ -0,0 +1,6 @@
+[build-system]
+requires = [
+    "setuptools>=42",
+    "packaging",
+]
+build-backend = "setuptools.build_meta"
```

Comment:
```suggestion

# We're not ready yet to build Spyder with the setuptools backend, but we're
# leaving this for the future.
# build-backend = "setuptools.build_meta"
```

It seems `check-manifest` (used to check the integrity of our tarballs) is trying to build Spyder using this info and that's why our tests are failing. If we comment it, then it runs without problems (tested locally).